snomed/com.b2international.snowowl.snomed.reasoner.server/src/com/b2international/snowowl/snomed/reasoner/server/classification/PersistChangesRemoteJob.java

```diff
@@ -129,6 +131,9 @@ private void lockBeforeChanges() {
 
 	private IStatus persistChanges(final IProgressMonitor monitor) throws CommitException {
 
+		//TODO: get this is a service or something
+		NamespaceAndMolduleAssigner allocator = new SourceConceptNamespaceAndModuleAssigner();
```

Comment:
This job needs the pluggable implementation more than `getResults` below; the constructor call is in `doPersistChanges`.